packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -251,19 +251,19 @@ export class GasEstimator {
 
   private _extractFastGasPrice(json: { [key: string]: any }, url: string): LondonGasData | LegacyGasData {
     if (url.includes("etherscan.io") || url.includes("polygonscan.com")) {
+      const isMainnet = url.includes("etherscan.io");
```

Comment:
Will optimism cause problems here? Their url is optimistic.etherscan.io. Or do we need to handle optimism at all?

Reply:
[Optimism is just defaulted](https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/helpers/GasEstimator.ts#L209)

Also I don't think the GasEstimator is used there